docs/usage/docker.md

```diff
@@ -248,7 +248,12 @@ module.exports = {
 
 #### AWS ECR (Amazon Web Services Elastic Container Registry)
 
-Renovate can authenticate with AWS ECR using AWS access key id & secret as the username & password, for example:
+Renovate can authenticate with AWS ECR using AWS credentials, login password or an authorization token.
+
+##### Using AWS account credentials
+
+To authenticate using AWS credentials, specify the access key id & secret as the username & password. If required
+define the session token also, for example:
```

Comment:
```suggestion
To authenticate using AWS credentials, you must use:

- the access key id as your username
- the secret as your pasword

If needed, you can set a session token, for example:
```

Please check if this suggestion is correct before applying it. The idea here is to put the terms next to their replacements, this should help users to use the right term in the right place.

How does a Renovate user know "If required define the session token also" applies to them? Do we need to explain this in the Renovate docs maybe?

Reply:
The suggestion looks good, thanks!

The token requirement depends on how the AWS credentials are issued...  It is possible to issue credentials both with and without requiring a session token.  It is typically managed by the AWS account admin.